worlds/openrct2/Client.py

```diff
@@ -0,0 +1,117 @@
+
+import logging
+import sys
+import asyncio
+import typing
+from CommonClient import CommonContext, get_base_parser, server_loop
+import Utils
+import re
+from .OpenRCT2Socket import OpenRCT2Socket
+
+if __name__ == "__main__":
+    print("\n\n\n\n\n\n==================================\n")
+    Utils.init_logging("TextClient", exception_logger="Client")
+# without terminal, we have to use gui mode
+gui_enabled = not sys.stdout or "--nogui" not in sys.argv
+
+logger = logging.getLogger("Client")
+
+class OpenRCT2Context(CommonContext):
+    tags = {"DeathLink"}
+    game = "OpenRCT2"
+    items_handling = 0b111  # receive all items for /received
+    want_slot_data = True 
+
+    def __init__(self, server_address: typing.Optional[str], password: typing.Optional[str]) -> None:
+        super().__init__(server_address, password)
+        self.gamesock = OpenRCT2Socket(self)
+        self.game_connection_established = False
+        #kivy.set_title("OpenRCT2 Client")
+
+    async def server_auth(self, password_requested: bool = False):
+        if not self.game_connection_established:
+            logger.info('Awaiting connection to OpenRCT2')
+            await self.gamesock.connected_to_game.wait()
+            
+        
+        if password_requested and not self.password:
+            await super(OpenRCT2Context, self).server_auth(password_requested)
+
+
+        await self.get_username()
+        await self.send_connect()
+
+    def on_package(self, cmd: str, args: dict):
+        # if cmd == "Connected":
+        #     self.game = self.game#slot_info[self.slot].game
+        print("PACKAGE!!!")
```

Comment:
I admire the enthusiasm but, maybe provide more info on the print? The `cmd` for instance? You print the args later anyway.